ome_zarr/writer.py

```diff
@@ -0,0 +1,112 @@
+"""Image writer utility
+
+"""
+import json
+import logging
+from pathlib import Path
+from typing import Any, List, Tuple, Union
+
+import dask.array as da
+import numpy as np
+import zarr
+
+from .io import parse_url
+from .reader import Node
+from .types import JSONDict
+
+LOGGER = logging.getLogger("ome_zarr.writer")
+
+
+def write_image(
+    path: str,
```

Comment:
I could imagine passing a `zarr.Group` here rather than path+name+group

Reply:
yes makes sense  (I realize the group argument here is not used). 
Do you think both way of specifying the output should co-exist or it's better to leave the job of creating the group to the user all the time?

Reply:
Let's start with leaving it to the user since there are so many options (local v. remote, etc.) but if we realize we're doing the same thing over and over again, we'll provide a helper.

Reply:
osund reasonable :)